ui/lib/components/accordion/accordion.dart

```diff
@@ -31,7 +31,14 @@ class AccordionItem {
   DivElement _headerWrapper;
   DivElement _bodyWrapper;
 
-  AccordionItem(this.id, this.headerElement, this.bodyElement, this._isOpen, {String dataId}) {
+  bool get isOpen => _isOpen;
+
+  void set id (String newID) {
```

Comment:
can we add an `id` getter as well? it might come in useful, and in the `Accordion` class you can continue to access the public member `id` rather than the (now) private one.